Python/pythonrun.c

```diff
@@ -1649,6 +1649,80 @@ PyOS_CheckStack(void)
 
 #endif /* WIN32 && _MSC_VER */
 
+#if defined(__APPLE__)
+
+#define MAC_OS_10_9 13
+#define MAC_OS_10_10 14
+#define MAC_OS_10_11 15
+
+#include <sys/utsname.h>
+
+size_t
+Get_Stack_Check_Size(const pthread_t *thread_self)
```

Comment:
Maybe inline this function in Init_Stack_Status()? Not sure if it's worth it to have a separated function.